_includes/link-list.html

```diff
@@ -1,14 +1,52 @@
 <dl class="link-list">
-{% for link in include.list %}
+{% for link in include.links %}
+
+{% if link.href_snapshot %}
```

Comment:
Selection of the link href is done in [LinkList](https://github.com/newtheatre/history-project/blob/master/_plugins/link_list.rb).

Apologies, this API isn't really documented (at all), but roughly sits inbetween the data in the show file (or other) and tweaks it before the this template gets ahold of it. This block should be removed and we just use `link.href` to refer to the resource.